keyboards/tada68/keymaps/sm0gunix/config.h

```diff
@@ -0,0 +1,4 @@
+#include "../../config.h"
```

Comment:
This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

And you should have a `#pragma once` at the top here.